doc/reference/instance_options.md

```diff
@@ -131,6 +131,13 @@ You can specify either which CPUs or how many CPUs are visible and available to
   Instances are re-balanced every time an instance starts or stops, as well as whenever a CPU is added to the system.
 
 ```{note}
+LXD supports live-updating the `limits.cpu` option.
+However, this only means that the respective CPUs are hotplugged.
```

Comment:
The part about the guest OS is only relevant for VMs, so perhaps something like:

> However, for VMs, this only means that the respective CPUs are hotplugged.
> Depending on the guest operating system, you might need to either restart the instance or complete some manual actions to bring the new CPUs online.

Reply:
Oh, okay - then this will need to go into the VM section below, actually. I thought it was for containers as well.

Reply:
Updated.